app/models/search_builder.rb

```diff
@@ -51,7 +54,15 @@ def convert_query_to_embedding_for_vector_search(solr_parameters)
     end
 
     # Vector embedding service returned embedding data.  Replace query with vector version.
-    # solr_parameters[:q] = "{!knn f=scopecontent_vector768i topK=9999}[#{query_vector.join(', ')}]"
-    solr_parameters[:q] = "{!vectorSimilarity f=scopecontent_vector768i minReturn=0.7}[#{query_vector.join(', ')}]"
+    # solr_parameters[:q] = "{!knn f=searchable_text_vector768i topK=9999}[#{query_vector.join(', ')}]"
+    solr_parameters[:q] = "{!vectorSimilarity f=searchable_text_vector768i minReturn=0.79}[#{query_vector.join(', ')}]"
+  end
+
+  def vector_search_enabled?
+    vector_search_override = blacklight_params[:vector_search]
+
+     return vector_search_override == 'true' if %w[true false].include?(vector_search_override)
+
+    CONFIG[:default_search_mode] == 'vector'
   end
```

Comment:
```suggestion
  def vector_search_enabled?
    return false if blacklight_params[:vector_search] == 'false'
    return true if blacklight_params[:vector_search] == 'true'
    CONFIG[:default_search_mode] == 'vector'
  end
```
Suggesting an optional refactor that might be slightly quicker to read!  I tried it locally and all of the tests still pass.  (Great test coverage, by the way!)